content/outcomes/secure-software-supply-chain/container-registry.md

```diff
@@ -0,0 +1,33 @@
+---
+title: How a Container Registry Can Help
+weight: 6
+layout: single
+---
+
+A container registry provides a single place to store and retrieve your container images. At a minimum, a container registry will support the [Docker Registry API](https://docs.docker.com/registry/spec/api/), the API that both Docker and Kubernetes expect to be supported to pull container images. They may either be a publicly hosted such as [Docker Hub](https://hub.docker.com/) or hosted privately such as [VMware Harbor](https://goharbor.io/).
+
+## Automated Security Scans
+
+A popular feature of container registry is the ability to automatically scan container images for known vulnerabilities. As a single container image can container many layers, including a base container image that you don't even maintain, it can be a huge challenge to stay on top of keeping your dependencies up to date. Because of this, you'll find many container registries offer the ability to scan your containers every time you upload one and report if there's a known vulnerability discovered. For example, Harbor relies on [Clair](https://github.com/quay/clair), an open-source solution that performs static analysis on container images. Of course, as new vulnerabilities are discovered, you can rerun these scans against your existing images. If an issue is found, you can configure your registry to block any requests to pull the image based on the severity.
+
+![An automated scan in Harbor finding vulnerabilities in our container](/images/outcomes/secure-software-supply-chain/harbor-scan.png)
+
+## Policy Enforcement
+
+Being the central point for gathering and distributing container images, the container registry has a lot of say in _who_ can push or pull an image. Every major container registry -- public or private -- supports requiring the user to be authenticated to interact with it. As such, the registry can then manage a list of actions the user (or group of users) can perform. For example, suppose we create a group called "developers" and they have access to an image that contains a collections of services that the require to perform their job. We can configure the registry to allow them to pull down the image to their laptop, but prevent them from pushing up new versions of the container. 
+
+The all contributes to one of the most fundamental ideas of security: the principal of least privilege. That is, the user should have access to only the actions and resources that they absolutely require.
+
+## Using a Container Registry in an Air-Gapped Environment
```

Comment:
Maybe this is the right place to mention the ability to proxy (and cache) other registries?